### ISSUE TITLE
No way to continue rebase on conflict

### ISSUE THREAD

**Issue by sZXZ** [ProjectBorealis/UE4GitPlugin#53]
You ca create conflict by naming assets in a same way
In this case user can remove their conflicting files with reset and only button missing is "continue rebase"

**Comment by mastercoms** [comment]
How about locking new assets?

**Comment by sZXZ** [comment]
Yeah locking helps. But if two users commit new file with a same name in the same time there is a conflict. I understand that this probably rare thing to have. But solution looks quite simple. Just to have a button for "continue rebase" if you want to discard your changes. If you want to keep them it's harder.

**Comment by mastercoms** [comment]
But if there is a lock before the new file is pushed, another user will not be able to make their own. I am just trying to avoid conflict states as much as possible. I do think in general the pull in editor is designed to overwrite local changes with server ones, but yes it would be good to handle the full process of resolution once it's done. So seems like there's 2 tasks here, is what I'm saying.

**Comment by sZXZ** [comment]
You can lock file that doesn't exist on a server? I wasn't aware that this is possible

**Comment by mastercoms** [comment]
Yes, it's allowed by GitHub at least. I'm not sure if it's part of the standard.
